devel/spice-server/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           meson 1.0
+
+name                spice-server
+version             0.14.3
+maintainers         {danchr @danchr} openmaintainer
+categories          devel
+platforms           darwin
+license             BSD
+description         Remote virtual machine server
+long_description    {*}${description}
+
+homepage            https://www.spice-space.org/
+master_sites        https://www.spice-space.org/download/releases/
+
+use_bzip2           yes
+
+checksums           rmd160  fe99d8d0db4b275b1d19dc9a7231144133c5bfa0 \
+                    sha256  551d4be4a07667cf0543f3c895beb6da8a93ef5a9829f2ae47817be5e616a114 \
+                    size    1504304
+
+patchfiles          meson-build.diff \
+                    no-werror.diff \
+                    pthread-setname-args.diff \
+                    no-msg-nosignal.diff \
+                    no-tools.diff \
+                    no-tests.diff
+
+configure.args-append \
+    -Dsasl=false \
+    -Dtests=no
+
+depends_build-append \
+    port:pkgconfig \
+    port:py38-six port:py38-parsing
+
+depends_lib-append \
+    port:lz4 \
+    port:spice-protocol \
+    port:glib2 \
+    port:libpixman \
```

Comment:
glib2 and libpixman are available in -devel flavors that are drop-in replacements for the non-devel flavors so please write these dependencies in `path:` style so that the -devel flavors could satisfy them. See any other port that depends on them for the proper syntax.